src/Domain/Package/Package.php

```diff
@@ -74,6 +74,9 @@ class Package {
    *     both of the "version" and "version_dev" key-value pairs to be used when
    *     the corresponding Drupal core version constraint is satisfied. Mappings
    *     are processed in order, and the first match wins.
+   *   - "is_company_package": (optional) Whether package is a company package
+   *     or not. Defaults to "true" but can be overridden. Anything else is
+   *     always false and cannot be overridden.
```

Comment:
I notice this `@param` documentation has gotten out of sync with the actual `packages.yml` schema. (It's missing `enable`.) I wonder if we should just remove the details altogether and point to `packages.yml` for details. A follow-up PR would be fine for any changes like this if you feel like it would be cleaner. I would be happy for you to create one and merge it without review.

Reply:
Okay, I will create a PR for this.

Reply:
Created PR https://github.com/acquia/orca/pull/403 and merging.